scripts/01_prep_inputs/0100-extract-inputs.R

```diff
@@ -668,26 +668,21 @@ hab_fish_indiv_prep3 |>
 hab_fish_indiv <- full_join(
   select(hab_fish_indiv_prep3,
          reference_number,
-         sampling_method,
```

Comment:
putting a note here 

I think Some other input must have been missing or incorrect for the full join not to function in 2023 if it is the same in all the past repos. ... We should try again with all the columns next repo and watch out for this cropping up downstream in this one. prob fine but worth a flag i think